sdk/src/androidTest/java/com/mapbox/maps/StyleLoadTest.kt

```diff
@@ -26,7 +26,7 @@ class StyleLoadTest {
   @UiThreadTest
   fun setUp() {
     rule.scenario.onActivity {
-      mapView = MapView(it)
+      mapView = MapView(it, MapInitOptions(it, plugins = listOf()))
```

Comment:
Can you please clarify a bit why having lifecycle enabled actually will trigger `styleLoadedCount` twice?

Reply:
With lifecycle, onStart method will be triggered automatically and since there is no style set, the default style will be loaded automatically.

Reply:
Oh, I see, good point.
Perhaps we may want separate tests then to check it? And add a comment that if we have lifecycle plugin enabled - we will actually check that `styleLoadedCount` is triggered twice (and drop a comment why it happens like this as this is not obvious).

Reply:
Wondering how this would affect our Map initialisation process, would it cause surprise to the customers that some event might be triggered multiple times?

Reply:
No event is triggered multiple times, simply because we [load style](StyleLoadTest.kt) again, plus the one we load automatically in onResume, it turns out style is load 2 times.

_Perhaps we may want separate tests then to check it? And add a comment that if we have lifecycle plugin enabled - we will actually check that styleLoadedCount is triggered twice_

That's a good point to verify this plugin, will add new tests for it.